src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27622.cs

```diff
@@ -0,0 +1,121 @@
+﻿#if WINDOWS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue27622 : _IssuesUITest
+	{
+		const string CollectionView = "CollectionView27622";
+		const string NoneRadioButton = "NoneRadioButton";
+		const string SingleRadioButton = "SingleRadioButton"; 
+		const string MultipleRadioButton = "MultipleRadioButton";
+
+		public Issue27622(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Windows CollectionView Keyboard Navigation";
+
+		[Test]
+		[Category(UITestCategories.Accessibility)]
+		[Category(UITestCategories.CollectionView)]
+		public void CollectionViewKeyboardNavigation()
```

Comment:
The test running:
![cv-keyboard-test](https://github.com/user-attachments/assets/8398386f-20a6-410f-b6e4-783f999a8832)